apps/snp_store/snp_store.cpp

```diff
@@ -938,103 +1128,177 @@ int detectSNPs(SNPCallingOptions<TSpec> &options)
-                    clipReads(fragmentStore,readClips,(unsigned)0,(unsigned)length(fragmentStore.alignedReadStore),options);
+                    // activate "average read quality"-mode for snp calling, low quality bases should be clipped anyway
+                    options.useBaseQuality = false;
+                    clipReads(fragmentStore,
+                              readClips,
+                              (unsigned)0,
+                              (unsigned)length(fragmentStore.alignedReadStore),
+                              options);
                 }
 
                 // check for indels
                 if (options.outputIndel != "")
                 {
-                    if(options._debugLevel > 1) ::std::cout << "Check for indels..." << std::endl;
-                    if(!options.realign) dumpShortIndelPolymorphismsBatch(fragmentStore, readCigars, fragmentStore.contigStore[0].seq, genomeNames[i], startCoord, currentWindowBegin, currentWindowEnd, indelFileStream, options);
+                    if (options._debugLevel > 1)
+                        ::std::cout << "Check for indels..." << std::endl;
+                    if (!options.realign)
+                        dumpShortIndelPolymorphismsBatch(fragmentStore,
+                                                         readCigars,
+                                                         fragmentStore.contigStore[0].seq,
+                                                         genomeNames[i],
+                                                         startCoord,
+                                                         currentWindowBegin,
+                                                         currentWindowEnd,
+                                                         indelFileStream,
+                                                         options);
                 }
 
-                // // check for CNVs
-                //              if (*options.outputCNV != 0)
-                //                  dumpCopyNumberPolymorphismsBatch(fragmentStore, genomeNames[i], startCoord, currentWindowBegin, currentWindowEnd, cnvFileStream, options);
+                //                // check for CNVs
+                //                if (*options.outputCNV != 0)
+                //                {
+                //                    dumpCopyNumberPolymorphismsBatch(fragmentStore,
+                //                                                     genomeNames[i],
+                //                                                     startCoord,
+                //                                                     currentWindowBegin,
+                //                                                     currentWindowEnd,
+                //                                                     cnvFileStream,
+                //                                                     options);
+                //                }
```

Comment:
As it seems there endeavours to add copy-number-variation detection or something similar to SNP-Store. I never touched these parts of the code myself, but all of them seem to be commented out. My guess woul be that the might come in handy, if someone wants to add this functionality in the future.